identity/src/main/java/com/android/identity/internal/Util.kt

```diff
@@ -1047,7 +1047,7 @@ object Util {
     }
 
     @JvmStatic
-    fun cborMapExtractMap(map: DataItem, key: String): DataItem {
+    fun cborMapExtractMap(map: DataItem, key: String): Map {
```

Comment:
I have a big PR which will delete this method (among others) so I'd prefer to not change this right now.